Golint install error in CI

cf: https://travis-ci.org/fabric8-services/fabric8-jenkins-proxy/builds/442149028?utm_source=github_status&utm_medium=notification

```
go get -u github.com/golang/lint/golint
package github.com/golang/lint/golint: code in directory /home/travis/gopath/src/github.com/golang/lint/golint expects import "golang.org/x/lint/golint"
make: *** [tools.timestamp] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -67,7 +67,7 @@ tools: tools.timestamp
 
 tools.timestamp:
 	go get -u github.com/golang/dep/cmd/dep
-	go get -u github.com/golang/lint/golint
+	go get -u golang.org/x/lint/golint
 	go get -u github.com/vbatts/git-validation/...
 	go get -u github.com/haya14busa/goverage
 	@touch tools.timestamp

--- a/internal/proxy/ui_requests.go
+++ b/internal/proxy/ui_requests.go
@@ -39,8 +39,7 @@ func (p *Proxy) handleJenkinsUIRequest(w http.ResponseWriter, r *http.Request, l
 
 		pci, osioToken, err := p.processToken([]byte(tj[0]), tjLogger)
 		if err != nil {
-			tjLogger.Errorf("Error processing token_json to get osio-token: %q", err)
-			http.Redirect(w, r, redirectURL.String(), http.StatusTemporaryRedirect)
+			p.HandleError(w, fmt.Errorf("Error processing token_json to get osio-token: %q", err), tjLogger)
 			return
 		}
 

